.github/workflows/main.yml

```diff
@@ -224,7 +224,7 @@ jobs:
     - run: cargo run -p run-examples
 
     # Build and test all features except for lightbeam
-    - run: cargo test --features test-programs/test_programs --all --exclude lightbeam --exclude wasmtime-c-api -- --nocapture
+    - run: cargo test --features test-programs/test_programs --all --exclude lightbeam -- --nocapture
```

Comment:
We are running c-api tests at line 249 below. Do we want to disable them there?

Reply:
Good eye -- done!